inlineplz/util/system.py

```diff
@@ -10,11 +10,82 @@
 from __future__ import unicode_literals
 
 
+import traceback
 import os
+import sys
+
+
+if sys.version_info >= (3, 5):
```

Comment:
Just a reminder that we'll need to merge these changes; most of the asyncio changes are in the \_\_init\_\_.py T_T